setup/pi/configure.sh

```diff
@@ -307,6 +325,21 @@ function configure_ifttt () {
     fi
 }
 
+function configure_telegram () {
+    if [ -n "${TELEGRAM_ENABLED+x}" ]
+    then
+        log_progress "Enabling Telegram"
+        {
+            echo "export TELEGRAM_ENABLED=true"
+            echo "export TELEGRAM_CHAT_ID=$TELEGRAM_CHAT_ID"
+            echo "export TELEGRAM_BOT_TOKEN=$TELEGRAM_BOT_TOKEN"
+            echo "export TELEGRAM_SILENT_NOTIFY=$TELEGRAM_SILENT_NOTIFY"
+        } > /root/.teslaCamTelegramSettings
```

Comment:
I realize this is the way the other notifications do it, but there's a simpler way to do it now:
 - because all the variables will have already been set by archiveloop (since it reads teslausb_setup_variables.conf when it starts), it is not necessary to create /root/.teslaCamTelegramSettings anymore
- instead of testing for the existence of /root/.teslaCamTelegramSettings and sourcing it in the send-push-message script, just check for TELEGRAM_ENABLED being set

Reply:
Can we go ahead and commit this for now? Knowing this also allows me to address issue #518 and merge the simplified approach with having a unique identifier per teslausb install.